dotnet/Capstone/Controllers/BirdController.cs

```diff
@@ -41,7 +43,7 @@ public List<Bird> ListAllBirds()
             
         }
 
-        [HttpGet("/birds/{id}")]
+        [HttpGet("/bird/{id}")]
```

Comment:
This API route returns 204 No Content instead of 404 not found when an id that has been deleted/doesn't exist yet is sent. Similarly, 204 is returned for invalid ids (<=0). Ideally that would be a bad request, but 404 would also work.

Reply:
Done